bindings/matrix-sdk-ffi/src/ruma.rs

```diff
@@ -362,6 +362,15 @@ impl TryFrom<MessageType> for RumaMessageType {
                         .info(content.info.map(Into::into).map(Box::new));
                 event_content.formatted = content.formatted_caption.map(Into::into);
                 event_content.filename = filename;
+
+                if let Some(audio) = content.audio {
+                    event_content.audio = Some(audio.into());
+                }
```

Comment:
Can't you just assign that always, something like:

```rust
event_content.audio = content.audio.map(Into::into)
```

Reply:
Oh right, Jorge told me something about mapping but I just blanked out 😅